Onboarding cleanup + misc changes

morebrownies opened the pull request:
**Changes**
- Fixed checkboxes in onboarding
- Fixed search icons in inputs in onboarding
- Made scrollbars a bit wider
- Bumped up the text size in the Discover toggle
- Typography fixes on the become a store modal

ilxwolf committed:
changes

ilxwolf committed:
font

ilxwolf committed:
typography fixes on become store modal

jjeffryes commented on js/templates/home.html:
```diff
@@ -49,9 +49,9 @@ <h1 class="fancy-heading fontSize20 vCentered js-discoverHeading"><%= polyglot.t
                 <span class="ion-person fontSize11 marginRight2 toggle-txt"></span>
                 <span class="toggle-txt"><%= polyglot.t('ShowOnlyFollowerListings') %></span>
               </a>
-              <% if(!localStorage.getItem('safeListingsWarningDissmissed')){ %>
```
Why did you remove this? If you do that, the discover helper will appear when it shouldn't.

morebrownies replied:
ah, was testing without it and forgot to re-enable it.

jjeffryes committed:
fix modal formatting, restore logic

jjeffryes committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into checkboxes

jjeffryes committed:
Add delay to follow after onboarding

- this is some hacky hackery, but it seems to fix the problem with
follow calls failing.

jjeffryes committed:
update
